src/helperFunctions/formatProjectDataHelper.js

```diff
@@ -12,6 +12,14 @@ export const formatProjectDataHelper = (project) => {
   ]
     .sort((a, b) => a.localeCompare(b))
     .join(', ')
-  const siteCount = [...new Set(project.records.map((record) => record.site_name))].length
-  return { projectName, formattedYears, countries, organizations, siteCount }
+  const surveyCount = project.records.length
+  const transects = project.records.reduce((acc, record) => {
+    return (
+      acc +
+      Object.values(record.protocols).reduce((acc, protocol) => {
+        return acc + protocol.sample_unit_count
+      }, 0)
+    )
+  }, 0)
```

Comment:
I agree. There's a lot going on in this function and it takes a while to begin to understand it.

Also, is there any way creating some of the consts can be combined? `map` and `reduce` are called several times each on `project.records`. That's a lot of repetition and it would be nice if it could be minimized.